pkg/module/metrics/latency.go

```diff
@@ -35,8 +35,8 @@ const (
 	TTL                             time.Duration = 500 * time.Millisecond
 	LIMIT                           uint64        = 100000
 	// Bucket size.
-	start = 0
-	width = 0.5
+	start = 0.5
+	width = 1
```

Comment:
Increasing the width will clump a lot of values into one bucket. I would recommend increasing the count to accommodate for results before `+inf`.